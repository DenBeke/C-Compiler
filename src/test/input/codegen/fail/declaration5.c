// Declared in other scope

#include <stdio.h>

int test(int a) {
    return a;
}

int b = a;

void main() {

}