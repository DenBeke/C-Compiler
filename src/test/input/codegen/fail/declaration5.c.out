[31m[ERROR] line 9: Use of undeclared 'a'[0m
